13_7_Lightning_Network_Implementations.md

```diff
@@ -0,0 +1,5 @@
+| Name  | Description | 
+| ------------- | ------------- |
```

Comment:
I think this is a good section to have, but you can keep it very brief. Just have a short section with how to download that one (and/or note if it's an option in Standup), and its advantages and disadvantages.